### DISCUSSION TITLE
What companies or projects are using clj-kondo?

### DISCUSSION THREAD

**Discussion by borkdude** [clj-kondo/clj-kondo#1397] [General]
Please share if your company or project is using clj-kondo. You're welcome to provide a little bit of detail about how you are using it well. No strings attached.

This has several purposes:

- Your company might be mentioned in a sheet of a presentation about clj-kondo. A link to a logo would be helpful.
- This issue would be a good way to notify you of any breaking changes in clj-kondo (which we will try to prevent).
- Your company is listed on [this](https://github.com/clj-kondo/clj-kondo/blob/master/doc/companies.md) page.
- It would help me a lot to understand who is using and how this project is being used.

If you would like to sponsor clj-kondo and related projects, that's be great:

https://opencollective.com/clj-kondo
https://github.com/sponsors/borkdude

I'm also available for [commercial support](https://blog.michielborkent.nl/oss-highlights-nov-dec-2021.html#enterprise-support) if you need that.

Similar discussions for:

- [babashka / SCI](https://github.com/babashka/babashka/discussions/1026)
- [nbb](https://github.com/borkdude/nbb/discussions/91)
- [SCI](https://github.com/babashka/sci/discussions/662)

**Comment by Deraen** [comment]
You can find Metosin logo here: https://github.com/metosin/media

**Comment by thenonameguy** [comment]
We are also using it at Ekata
https://ekata.com/
https://ekata.com/assets/ekata-logo-white.svg

Thanks for this great piece of software!

**Comment by liquidz** [comment]
We are also using clj-kondo. Thanks for your great job!

* TOYOKUMO,Inc.
  * https://toyokumo.co.jp
* Logo
  * https://toyokumo.co.jp/wp-content/uploads/2019/09/toyokumo_logo.png

**Comment by Kynde** [comment]
We are also using clj-kondo at Leanheat. Nice linter! Keep up the good work! See you in ClojuTre.
https://leanheat.com/
https://leanheat.com/wp-content/uploads/2019/05/LeanheatDanfossLogoMarginBigFix.png

**Comment by Juholei** [comment]
Solita is using clj-kondo in multiple projects. Thank you for your work!
https://www.solita.fi/en/
Logo: https://user-images.githubusercontent.com/5576385/65430683-f6a77480-de20-11e9-858e-538be8944e6c.png

**Comment by scramjet** [comment]
Systematic is using it. https://systematic-app.com

Thanks! Finding it very useful in combination with flycheck on Emacs.

**Comment by gwena** [comment]
Ticketless.ai is using clj-kondo. Thanks a lot. Nice integration with Emacs.
https://ticketless.ai
Logo: https://ticketless.ai/wp-content/uploads/2019/08/logo.svg

**Comment by pbalduino** [comment]
Klarna
https://klarna.com
Logo: https://www.klarna.com/se/wp-content/themes/klarna/assets/images/icons/klarna.svg

Thank you

**Comment by chorin1** [comment]
Using this lovely linter on AppsFlyer
https://appsflyer.com

Thanks for building this great tool!

**Comment by DanBunea** [comment]
Happy user from Codurance, Barcelona 
https://www.codurance.com

**Comment by telekid** [comment]
You've got some happy users over here at Flexport!

https://www.flexport.com

🚢

**Comment by ericdallo** [comment]
Some users from Nubank!

https://www.nubank.com.br

**Comment by pfeodrippe** [comment]
Here at Zen Finance we use it for most of our projects o/ thank you!

**Website**: https://www.zenfinance.com.br/

**Logo**: https://www.zenfinance.com.br/imgs/logo.png?8e502a4

**Comment by nenadalm** [comment]
Imatic is using clj-kondo

website: https://www.imatic.cz/
logo: https://avatars1.githubusercontent.com/u/568033?s=200&v=4

**Comment by yannvanhalewyn** [comment]
Brightin is using CLJ-kondo!

website: https://www.brightin.nl/
logo: https://user-images.githubusercontent.com/11170603/69530070-b4f68f80-0f71-11ea-9d95-052703cd0367.png

**Comment by kwrooijen** [comment]
Mediquest uses this, thanks for your work!

Website: https://home.mediquest.nl/
Logo: https://home.mediquest.nl/wp-content/uploads/2019/07/mq-logo-rgb-online.png

**Comment by jeroenvandijk** [comment]
Adgoji is using clj-kondo. Thank you!
website: https://www.adgoji.com/

Logo [here](https://drive.google.com/file/d/1VlUhnrodGmZZd8bQl-PFgJPBdJTf_H4M/view)

**Comment by sskorokhodov** [comment]
We use clj-kondo at [SUPREMATIC](https://suprematic.de). Thank you!

**Comment by borkdude** [comment]
Added all these companies to the list here: https://github.com/borkdude/clj-kondo/blob/master/doc/companies.md

**Comment by bobbicodes** [comment]
If you are still taking names, we are using clj-kondo's analysis output in our tooling for the new Exercism platform, in order to provide automated feedback for students learning Clojure and Babashka.

[Exercism website](https://exercism.io/)
[Exercism logo](https://avatars0.githubusercontent.com/u/5624255)

**Comment by gwena** [comment]
Ticketless.ai is now Kaizo (https://kaizo.com/) and still using clj-kondo. Thanks a lot @borkdude

**Comment by borkdude** [comment]
I'm still taking names and updating them here:  https://github.com/borkdude/clj-kondo/blob/master/doc/companies.md

Thanks a lot for using clj-kondo!

**Comment by rubiagatra** [comment]
We are from https://zero-one-group.com using clj-kondo for our daily workflow. the logo https://zero-one-group.com/wp-content/uploads/2019/12/ZeroOneGroup_Black_500x500_Padded.png

**Comment by borkdude** [comment]
@raymcdermott on Slack:

FYI we at Vouch are now using clj-kondo ... please feel free to add us to your list

**Comment by laurio** [comment]
Happy user from Wunderdog, Helsinki
https://wunderdog.fi/
Thank you!

**Comment by pdenno** [comment]
We like clj-kondo at the US National Institute of Standards, https://www.nist.gov/

**Comment by bfontaine** [comment]
We’re using it at Oscaro (https://www.oscaro.com/).

**Comment by jplaza** [comment]
Thanks @borkdude for building this great tool! We've been using it for a long time now at Datil ([https://datil.com](https://datil.com))

**Comment by tanzoniteblack** [comment]
[Yummly](https://www.yummly.com) is using it and loving it!

http://assets.whirlpoolcorp.com/logos/Yummly-Logo-2018.png

**Comment by nathanmarz** [comment]
[Red Planet Labs](https://redplanetlabs.com/) is now using it. Excellent tool.

https://redplanetlabs.com/images/red-planet-logo.png

**Comment by rdgd** [comment]
[Splash Financial](https://www.splashfinancial.com/) is using clj-kondo.

[Splash logo](https://www.splashfinancial.com/wp-content/uploads/2021/05/logo.svg)

**Comment by borkdude** [comment]
I updated the companies page [here](https://github.com/clj-kondo/clj-kondo/blob/master/doc/companies.md)!

Sponsoring companies get listed first and you will be able to add a direct hiring link too if applicable.

I have quit my job and will be able to work more on clj-kondo (and babashka) in the future. Any additional sponsoring will help me keep doing that.

Thanks for using clj-kondo. If your company is using clj-kondo, please reach out. 

I converted the issue into a discussion so we can have a chat if I (or someone else) would like to know more details about how you are using clj-kondo.

**Comment by slipset** [comment]
[ardoq](https://www.ardoq.com) is using clj-kondo

**Reply by borkdude** [reply]
Thank you! Added to the list.

**Comment by dharrigan** [comment]
[healthunlocked.com](https://healthunlocked.com) :-)

**Comment by n2o** [comment]
Hey, I am one of the founders of [schnaq](https://schnaq.com) ([Logo](https://s3.schnaq.com/schnaq-common/logos/schnaq.svg)), we are a full-clojure company and love clj-kondo! It enriches our IDE and is always a Job in our CI/CD Pipeline. Thanks for your work! Since last week we are also using the analysis-tools to find more unused vars or missing docstrings 👍

**Comment by yannvanhalewyn** [comment]
[Pilloxa](https://www.pilloxa.com/) is using clj-kondo! ❤️

**Comment by IGJoshua** [comment]
[IRIS.TV](https://iris.tv) is using clj-kondo!
I'll see if I can get a good quality version of the logo.

**Comment by LouDnl** [comment]
We use it at [Thorax](https://www.thorax.nl) in our API connection services for healthcare.

**Comment by NoahTheDuke** [comment]
[Crossbeam](https://www.crossbeam.com/) is a heavy user. We gate our backend CI behind clj-kondo's checks passing.
